src/common/util.h

```diff
@@ -68,7 +68,11 @@ std::vector<std::string> TokenizeRedisProtocol(const std::string &value);
 
 void ThreadSetName(const char *name);
 int aeWait(int fd, int mask, uint64_t milliseconds);
-uint64_t GetTimeStampMS();
-uint64_t GetTimeStampUS();
+template <typename Duration = std::chrono::seconds>
+auto GetTimeStamp() -> Duration::rep {
```

Comment:
```suggestion
auto GetTimeStamp() {
```

Reply:
Looks we can't put the function body in header file, it will cause duplicate symbol conflicts.